ee-9/galleon-content/src/main/resources/layers/standalone/observability/layer-spec.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="observability">
+    <dependencies>
+        <layer name="health" optional="true" />
```

Comment:
Shouldn't micrometer be here?

Originally the 'observability' concept (not the layer) meant logging, metrics, health, distributed tracing. We created a layer around that concept. We didn't put logging in it, because it was always present anyway. (Which isn't conceptually valid and something to reconsider for WF 28.) And then tracing didn't make the layer because early on we got feedback that MP OT added runtime overhead even if not really sending any spans. (Which we should reexamine for WFP 28, i.e. whether we should put OTel in WFP's 'observability' layer.) Which left us with a layer for providing metrics in some form and one for health.

So I think with Micrometer in WFP, it takes the place of WF Metrics in providing the metrics part of observability.

Reply:
Updated.